cellcounter/cc_kapi/factories.py

```diff
@@ -22,13 +22,37 @@ class Meta:
     is_superuser = False
 
 
-class KeyboardFactory(factory.django.DjangoModelFactory):
+class DefaultKeyboardsFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = DefaultKeyboards
+
+
+class DefaultKeyboardFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = Keyboard
+        strategy = factory.BUILD_STRATEGY
+
+    class Params:
+        mappings = None
+
+    id = 0
+    user = None
+
+
+class DefaultKeyMapFactory(factory.DjangoModelFactory):
```

Comment:
What is the reason for having a 'DefaultKeyMapFactory' and a 'KeyMapFactory' these are essentially the same.